test/test_kaldi_dirs.py

```diff
@@ -220,6 +221,7 @@ def test_ok_on_file_singlechannel_sph_source_type(tmp_path, channel):
     assert list(out.keys()) == [channel]
     assert out[channel].startswith("sph2pipe")
     assert "nonexistent.sph" in out[channel]
+    assert "{" not in out[channel]
```

Comment:
Just curious to know what's this `assert` verifying?

Reply:
it's just detect someone has goofed up and didn't fully formatted the f-string (or forgot to add f"")